recipes/breakfest/arabic_omelette.md

```diff
@@ -0,0 +1,35 @@
+# Arabic Omellete
+
+![Difficulty Badge](https://img.shields.io/badge/Difficulty-10%25-brightgreen.svg)
+![Speed Badge](https://img.shields.io/badge/Speed-10min-brightgreen.svg)
+![Reception Badge](https://img.shields.io/badge/Reception-Very%20Positive-brightgreen.svg)
+![Cost Badge](https://img.shields.io/badge/Cost-Very%20Cheap-brightgreen.svg)
+![Vegetarian Friendly Badge](https://img.shields.io/badge/Vegetarian-True-brightgreen.svg)
+![Vegan Friendly Badge](https://img.shields.io/badge/Vegan-False-red.svg)
+
++ **First cooked by:**
+[@Karyum](https://github.com/karyum)
+
++ **Date:** Week 5 FACN2
++ **Author of Recipe:**
+#MrsSaliba
+
+### Ingredients
+- 2 small eggs.
+- a handful of parsley.
+- a handful of green/spring onions.
+- zaatar to taste.
+- salt and pepper
+
+### Preparation
+
+1. Chop the parsley and the green/spring onions into small pieces.
+2. add the everything into a small bowl and stir it with a **fork**.
+3. heat up a pan with a bit olive oil and wait for 1 min untill it heats up.
+4. add all of the mix into the heated pan.
+5. from here it's self preference on how much you want the Omellete to be cooked, I would say flip it when gets golden-red.
+
+Tips:
+- The more you stir it the fluffier the Omellete is going to get.
+- These are the base ingredients you could always get creative and add new ingredients on top of the base.
```

Comment:
Might be a nice idea to add in an ingredient combination that has worked well for you? :+1:

Reply:
I literally said i want people to be creative i should not give hints on what to add

Reply:
Fair point :shipit: